_data/domains.yml

```diff
@@ -55,11 +55,11 @@ websites:
       doc: http://blog.dnsimple.com/2012/08/protect-your-dnsimple-account-with-two-factor-authentication-from-authy/
 
     - name: Domain.com
-      url: https://domain.com
+      url: http://www.domain.com/
```

Comment:
The proofer failed on it apparently. I can change it back and see if it still fails.